CI failure due to traceback in integration tests:

```
Traceback (most recent call last):
  File "/tmp/ansible_y3Zf75/ansible_module_ec2_vpc_net.py", line 391, in <module>
    main()
  File "/tmp/ansible_y3Zf75/ansible_module_ec2_vpc_net.py", line 310, in main
    vpc_obj = get_vpc(module, connection, vpc_id)
  File "/tmp/ansible_y3Zf75/ansible_module_ec2_vpc_net.py", line 201, in get_vpc
    catch_extra_error_codes=['InvalidVpcID.NotFound'],
TypeError: backoff() got an unexpected keyword argument 'retries'
```

--- a/lib/ansible/modules/cloud/amazon/ec2_vpc_net.py
+++ b/lib/ansible/modules/cloud/amazon/ec2_vpc_net.py
@@ -197,7 +197,7 @@ def vpc_exists(module, vpc, name, cidr_block, multi):
 def get_vpc(module, connection, vpc_id):
     try:
         vpc_obj = AWSRetry.backoff(
-            delay=1, retries=5,
+            delay=1, tries=5,
             catch_extra_error_codes=['InvalidVpcID.NotFound'],
         )(connection.describe_vpcs)(VpcIds=[vpc_id])['Vpcs'][0]
     except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:

--- a/lib/ansible/modules/cloud/amazon/ec2_vpc_net.py
+++ b/lib/ansible/modules/cloud/amazon/ec2_vpc_net.py
@@ -227,7 +227,10 @@ def update_vpc_tags(connection, module, vpc_id, tags, name):
         if tags != current_tags:
             if not module.check_mode:
                 tags = ansible_dict_to_boto3_tag_list(tags)
-                connection.create_tags(Resources=[vpc_id], Tags=tags)
+                vpc_obj = AWSRetry.backoff(
+                    delay=1, tries=5,
+                    catch_extra_error_codes=['InvalidVpcID.NotFound'],
+                )(connection.create_tags)(Resources=[vpc_id], Tags=tags)
             return True
         else:
             return False

